packages/common-utils/src/clickhouse.ts

```diff
@@ -340,6 +340,7 @@ const localModeFetch: typeof fetch = (input, init) => {
     .atob(auth.substring('Bearer'.length))
     .split(':');
   delete init.headers?.['Authorization'];
```

Comment:
nit: probably worth refactoring into its own function in the future (not sold we need to today)